build_tools/pkgci/external_test_suite/attention_and_matmul_spec.mlir

```diff
@@ -636,8 +636,8 @@ module attributes { transform.with_named_sequence } {
          workgroup_size = [320, 1, 1] subgroup_size = 64,
           {mma_schedule = #iree_gpu.mma_schedule<
               intrinsic = #iree_gpu.mma_layout<MFMA_F32_16x16x16_F16>,
-              subgroup_m_count = 1, subgroup_n_count = 5>
-           , reorder_workgroups = "transpose"}>
+              subgroup_m_count = 1, subgroup_n_count = 5>,
+           gpu_pipeline_options = #iree_gpu<gpu_pipeline_options[ReorderWorkgroupsTranspose]>}>
```

Comment:
Will this work with `swizzle` that takes an additional parameter for the tile size? If we don't care much about swizzle specifically, does the new attribute support passes with more general options?